_sources/canasta/getMediawikiSettings.php

```diff
@@ -21,6 +21,12 @@ public function __construct() {
 			false,
 			true
 		);
+		$this->addOption(
+			'version',
+			'',
```

Comment:
Please document this parameter, I don't fully understand it, and how is it different from the `versions` parameter?